core/templates/dev/head/css/oppia.css

```diff
@@ -60,6 +60,12 @@
   100% { opacity: 1; }
 }
 
+@-webkit-viewport{width:device-width}
```

Comment:
Hmm, I think I know less about this than you do at this point, so I'll defer to you to make the call on this one :)